ladybug/stat.py

```diff
@@ -191,13 +176,58 @@ def _import_data(self):
                 r"Monthly Statistics for Wind Speed[\s\S]*Daily Avg(.*)")
             for direction in self._wind_dir_names:
                 re_string = r"Monthly Wind Direction %[\s\S]*" + direction + r"\s(.*)"
-                self._monthly_wind_dirs.append(self._regex_parse(re_string))
+                dirs = self._regex_parse(re_string)
+                if dirs != []:
+                    self._monthly_wind_dirs.append(dirs)
+
+            # check to see if all monthly data is there
+            if self._monthly_db_range_50 != [] and self._monthly_wb_range_50 != [] \
```

Comment:
should this be a separate property? What if one changes the value for any of the lists and items? I'm not sure if that is a possibility but it seems that you want to wrap this in a separate method that gets called every time that you need to check if monthly value is found.

Reply:
Everything that it is checking is is a hidden attribute so I am not expecting people to set those properties (thereby needing a re-check).  If you think that some people might try setting the hidden attributes, I can make it it's own property.  It might just be a bit inefficient since it will get called every time someone calls one of the other properties instead of being called once but I can see this inefficiency would be worth it if people will be setting hidden attributes regularly.

Reply:
I just implemented it as a property anyway.